compliance_checker/cf/cf.py

```diff
@@ -1910,7 +1904,7 @@ def check_calendar(self, ds):
                                     valid_calendar,
                                     '§4.4.1 Time and calendar',
                                     reasoning)
-                ret_val.append(result)
+                    ret_val.append(result)
```

Comment:
At this point in logic, we've returned already if the attribute doesn't exist.  I agree there's no Result for that, because we didn't check anything.  Specificaly, at this code juncture, it only is returning a Result if `not v.calendar in valid_calendars`, and that's incorrect.  We've found the calendar attribute, tell me if it's valid or not.